ctk/features/composite.feature

```diff
@@ -40,19 +38,16 @@ Feature: Composite Task
       namespace: default
       name: composite-sequential
     do:
-      setRGB:
-        execute:
-          concurrently:
-            setRed:
-              set:
-                colors: ${ .colors + ["red"] }
-            setGreen:
-              set:
-                colors: ${ .colors + ["green"] }
-            setBlue:
-              set:
-                colors: ${ .colors + ["blue"] }
-          compete: true
+      compete:
```

Comment:
That is a shadow change. Please avoid doing so in a single PR, as it hasn't been approved.

Reply:
I am not sure what you mean by "shadow change"?

We can keep the composite task as one using `execute` but it was agreed upon to remove `execute`. I mean as you said it is uglier with `execute`

Reply:
I mean it's a change I overlooked and did not think was a part of this PR. It's my bad.